### ISSUE TITLE
Console freezes on any command

### ISSUE THREAD

**Issue by perrygeo** [yhat/rodeo#342]
Fresh install of 2.0.0 using the dmg install on OSX 10.10.5

Typing any command into the console and it immediately freezes rendering it unusable. 

![screen shot 2016-06-15 at 7 48 02 am](https://cloud.githubusercontent.com/assets/1151287/16078644/10b489be-32ce-11e6-9972-23c0c3dfabff.png)

I'm able to restart and type a single command but it always freezes after I hit enter.

**Comment by coristig** [comment]
Hey @perrygeo, there's a log file called Rodeo.log in your home directory ($ open ~/Rodeo.log)

Can you paste the output of that file here?

Appreciate the feedback

**Comment by TakenPilot** [comment]
Yeah, those logs that @coristig mentioned will have useful stuff in there.

That said, there are probably two potential causes of this:
- It's having trouble running the instance of python you pointed it to (or that was auto-detected).  The solution would be to point it to a better python in the Preferences menu.
- It's running something in your .bash_rc or .bash_profile that's failing, so the console never starts.

Both of these things will be noted in the log.  I'm trying to think of ways to expose these kinds of error in the UI.

**Comment by ben519** [comment]
Also make sure you have jupyter installed. [How I set up Rodeo,](http://stackoverflow.com/a/37713789/2146894) in case you're interested.

**Comment by narenreddy** [comment]
I just installed 2.0 version and its freezing at console. My previous version is working like a charm. Its a biggest BUG. Not sure how come this version been released with kind of bug. May be its specific to my Mac OS version. I'm on Mac OS X version 10.11.15(15F34).

**Comment by perrygeo** [comment]
Steps to recreate
```
$ mkvirtualenv rodeo -p python3.5
(rodeo)$ pip install -U pip
(rodeo)$ which python
/Users/mperry/env/rodeo/bin/python
```
Then set "Python command" in the Rodeo settings, restart Rodeo, type anything into the console `a = 1` and hit enter. The console is completely frozen.

The last line in my `~/rodeo.log` file is

```
2016-06-16T12:35:23.575Z - error: Applications/Rodeo.app/Contents/Resources/app/node/kernels/python/client:: stderr.data '    from jupyter_client import manager\nImportError: No module named \'jupyter_client\'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "/Applications/Rodeo.app/Contents/Resources/app/node/kernels/python/start_kernel.py", line 7, in <module>\n    from IPython.kernel import manager\nImportError: No module named \'IPython\'\n'
```

Looks like all it needs is Ipython, `(rodeo)$ pip install jupyter` got me a working environment.

For users unfamiliar with Rodeo or who may consider Rodeo an alternative to Jupyter, it's not obvious that it is an implicit dependency. Rodeo should check for it's dependencies in the current python environment and explicitly state any missing packages in an error message rather than freezing.

**Comment by narenreddy** [comment]
Not sure why there is a dependency on Jupyter. Anyway, I gavotte following command and still the hanging problem is there. Now RODEO is not at all working. Strange upgrade.

$pip install jupyter
Requirement already satisfied (use --upgrade to upgrade): jupyter in /anaconda/lib/python3.5/site-packages

**Comment by TakenPilot** [comment]
Here is an example in the logs from Greg:
```
2016-06-16T12:24:52.680Z - info: Applications/Rodeo.app/Contents/Resources/app/node/kernels/python/steve-irwin:: findPythons { pythonOptions: { cmd: [32m'python'[39m, shell: [32m'/bin/bash'[39m, label: [32m'python'[39m },
  rejected: [32m'/Users/glamp/.bashrc: line 137: brew: command not found\n'[39m }
```

I'm trying to figure out why the PATH is behaving like this on some machines, but this is definitely a potential source.  If we can't start a shell, then no commands will work.

@narenreddy Jupyter is IPython, which is what this IDE connects to.

**Comment by narenreddy** [comment]
THE Problem is SOLVED. 

set your python path in Rodeo --> Preferences --> Python --> Python Command --> //anaconda/bin/python

**Comment by TakenPilot** [comment]
I'm going to build version 2.0.1 today that will add more to the PATH of OSX machines before bashrc, bash_profile and profile are run, and that should solve most of these issues.  Based on what people have said and what I've read, I think all of these problems are coming from launchd on OSX not having the same base PATH as the terminal, so the startup scripts are failing.

I'm just getting people around here to test it before I put it up.

**Comment by TakenPilot** [comment]
This fix is included in 2.0.1.

**Comment by jmaralc** [comment]
Same problem here with the 2.0.10 but @narenreddy answer was the right solution. 

Thanks a lot in deed is a nice IDE for python
